index.js

```diff
@@ -98,7 +98,7 @@ class SitemapStream extends EventEmitter {
 
     const isWritten = this.writer.write(`${loc}${lastMod}${changeFreq}${priority}${mobile}`);
 
-    if (isWritten) this.nbInjectedUrls++;
+    this.nbInjectedUrls++;
```

Comment:
@irnc sorry for the delay, since [the doc](https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback) advise us to stop writing when it returns false, i wasn't really sure about how to handle that part, but you are right, we should be as flexible as the Stream API is, plus i didn't lock the ability to write until it got drained, so the choice wasn't entirely taken. Thanks for your PR !